Update Kafka consumer integration documentation with configuration ecample (#20745)
DataDog/integrations-core: kafka_consumer/README.md

### Installation

> The Agent's Kafka consumer check is included in the [Datadog Agent][2] package. No additional installation is needed on your Kafka nodes, but configuration is still required.

Reviewer: ```suggestion
The Agent's Kafka consumer check is included in the [Datadog Agent][2] package. No additional installation is needed on your Kafka nodes. However, you still need to configure the integration.
```

1. Edit the `kafka_consumer.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][3]. See the [sample kafka_consumer.d/conf.yaml][4] for all available configuration options.

> Here's a barebones example configuration:

Reviewer: ```suggestion
Here's a basic example configuration to help you get started:
```

```yaml
init_config:

instances:
  - kafka_connect_str: kafka:29092 # Replace with your Kafka cluster address
    monitor_unlisted_consumer_groups: true
```

> 

Reviewer: ```suggestion
You can find more detailed configuration options in the sample configuration file linked above.
```
